lucida/djinntonic/dig/Makefile

```diff
@@ -23,7 +23,7 @@ $(TARGET): $(OBJECTS)
 	$(CXX) -Wall $(CXXFLAGS) -c $< -o $@
 
 start_server: 
-	gnome-terminal -x bash -c "./DIGServer $(filter-out $@,$(MAKECMDGOALS)); read -n1"
+	./DIGServer
 
 start_test: 
 	gnome-terminal -x bash -c "./run-test.sh $(filter-out $@,$(MAKECMDGOALS)); read -n1"
```

Comment:
I see your point but I think in that case there should be a better way to test. can you open an issue about moving `start_test` as unittest, similar to what we do internally? Look at the infrastructure we use